experiments/chat/chat/templates/chat/message_update.html

```diff
@@ -1,5 +1,5 @@
-{#<turbo-stream action="append" target="message-list">#}
-{#  <template>#}
-{#    {% include 'chat/message.html' with message=message only %}#}
-{#  </template>#}
-{#</turbo-stream>#}
+<turbo-stream action="append" target="messages">
```

Comment:
will there be a duplicate message here on the sender's list? One with `from http` and one with `from WS`?

Reply:
Yes indeed, messages will be duplicated. I guess we should have a nice way to choose between getting the turbo-stream from WS or HTTP.
Having an action="replace-or-append" in Turbo Streams would be awesome so a user without WS would still get his own message via HTTP while a user with WS would get messages from every user 🙈